pkgs/tools/misc/fbmark/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "fbmark";
+  version = "0.3";
+
+  src = fetchFromGitHub {
+    owner = "caramelli";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0n2czl2sy1k6r5ri0hp7jgq84xcwrx4x43bqvw1b4na99mqhyahn";
+  };
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  meta = with lib; {
+    description = "Linux Framebuffer Benchmark";
+    homepage = "https://github.com/caramelli/fbmark";
+    license = licenses.gpl3Only;
```

Comment:
```suggestion
    license = licenses.gpl3Plus;
```

Reply:
@figsoda from where did you get that?

I opened an issue and asked for clarification: https://github.com/caramelli/fbmark/issues/2

Reply:
the license headers of the fb_*.c files

https://github.com/caramelli/fbmark/blob/c6eddced28efd544db10cad1d87e760ad830cdc0/fb_mandelbrot.c#L7-L8

Reply:
Thank you! Didn't look there.